Sources/PrimerSDK/Classes/Core/Analytics/AnalyticsService.swift

```diff
@@ -92,11 +92,11 @@ extension Analytics {
             Promise { seal in
                 let events = storage.loadEvents()
                 sync(events: events, isFlush: true)
-                .done {
-                    seal.fulfill()
-                }.catch { error in
-                    seal.reject(error)
-                }
+                    .done {
```

Comment:
This formatting is off

Reply:
<img width="469" alt="Screenshot 2024-01-29 at 18 47 49" src="https://github.com/primer-io/primer-sdk-ios/assets/137198782/b4b17052-f812-499c-bee8-f28d133f1462">
it is not in Xcode 😕